### ISSUE TITLE
Output of BAGAN for CIFAR-10 is just solid colors.

### ISSUE THREAD

**Issue by siddharthdivi** [IBM/BAGAN#5]
@stevemart @ova-mariani 
![plot_class_0_epoch_0](https://user-images.githubusercontent.com/25533342/52256725-e009e700-293d-11e9-8e39-36181a489466.png)

I've tried to modify the BAGAN architecture for CIFAR-10, but unfortunately, the output of the BAGAN is random. I even used the same learning rate as mentioned in ACGAN. I've tried out different learning rates as well but to no avail. 

I have made the requisite changes in all the related files to enable BAGAN to be trained on a multi-channel input. Could you guys mention the hyper-parameters that you used for CIFAR-10 so that I may be able to get to the bottom of this anomaly?

**Comment by ova-mariani** [comment]
Hi @siddharthdivi ,

I have just pushed code that should enable you to execute training on CIFAR-10. Pass the flag '-D CIFAR10' when calling the training script.
We were training the model until convergence, I think we were using 100 epochs for both autoencoder initialization and adversarial training. That is probably an overshoot, you may stop earlier.

**Comment by siddharthdivi** [comment]
Thanks @ova-mariani
